quirks/shared-credentials.json

```diff
@@ -255,6 +255,16 @@
             "wikimania.wikimedia.org"
         ]
     },
+    {
+        "from": [
+            "www.seek.com.au",
```

Comment:
Just being a little nit picky here but would you mind sorting it based on the actual website domain in this case, according to `seek`, as the `www` can be typical for many sites. Happy to merge after that minor change, thanks!

Reply:
That's fair, thanks. I've rebased and addressed this.

Reply:
I apologize 😬, I gave you the wrong advice, the order I proposed is preventing it from passing the shared credentials sort order linting which I believe you had right the first time.

Reply:
Oh 😄 I liked it better too. Ran the lint script itself and it sorted it for me, should be good now